testament/categories.nim

```diff
@@ -169,16 +169,8 @@ proc dllTests(r: var TResults, cat: Category, options: string) =
 # ------------------------------ GC tests -------------------------------------
 
 proc gcTests(r: var TResults, cat: Category, options: string) =
-  template testWithNone(filename: untyped) =
-    testSpec r, makeTest("tests/gc" / filename, options &
-                  " --gc:none", cat)
-    testSpec r, makeTest("tests/gc" / filename, options &
-                  " -d:release --gc:none", cat)
```

Comment:
--gc:none is now not being tested at all here despite what the commit msg `remove 'gc:none' and 'refc' without 'd:useRealtimeGC' from gc tests` seems to imply

Reply:
I don't understand your comment.

Reply:
maybe i misread your commit message but regardless, after your PR, `--gc:none` is not being tested anymore. Is that becaus `--gc:arc` is to replace it?

Reply:
--gc:arc is replacing it an also because ``--gc:none`` tests are hardly ever broken. Chance for regressions is minor.